app/models/registry.rb

```diff
@@ -130,12 +130,12 @@ def create_namespaces!
       name:   "portus_global_team_#{count}",
       owners: User.where(admin: true),
       hidden: true)
-    Namespace.create!(
-      name:     "portus_global_namespace_#{count}",
-      registry: self,
-      public:   true,
-      global:   true,
-      team:     team)
+    Namespace.create!(name:        "portus_global_namespace_#{count}",
+                      registry:    self,
+                      public:      true,
+                      global:      true,
+                      description: "The global namespace for the registry, #{Registry.name}.",
+                      team:        team)
```

Comment:
Why all the extra spacing? Can't you do it in the same way as you did in the `app/models/user.rb` file?